lib/src/front_end/delimited_list_builder.dart

```diff
@@ -431,7 +430,30 @@ class DelimitedListBuilder {
         case BlockFormat.collection:
           collections.add(i);
         case BlockFormat.invocation:
```

Comment:
[optional] Consider reordering the cases by behavior - cases where the element is collected vs ignored.

Reply:
I thought about that, but I (for perhaps not justifiable reasons) tend to keep cases in the same order that the enum defines the values. That way every switch on that same type has the same order. I'm not sure if that's actually better or not, but it made sense to me at the time.